packages/edit-site/src/components/sidebar-navigation-data-list/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { __experimentalVStack as VStack } from '@wordpress/components';
+/**
+ * Internal dependencies
+ */
+import DataListItem from './data-list-item';
+
+export default function SidebarDetails( { details } ) {
```

Comment:
The naming of the file and the above `DataListItem` component feel a bit ambiguous to me. Are we going to reuse them elsewhere? Should we move them internally to `SidebarNavigationScreenPage`?

Reply:
Agree. I suspect it's specific to display individual entity properties, e.g., page/template. @SaxonF can confirm.

We can rename in a follow up if so.